### PR TITLE
i18n(pt-BR): Add `publish-to-npm.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

PT-BR translation for `publish-to-npm.md`.

### CODE REVIEW

File: src/pages/pt-BR/guides/publish-to-npm.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+| Análise     | `analytics`                              |
+| CMS         | `cms`, `database`                        |
+| CSS + UI    | `css`, `ui`, `icon`, `icons`, `renderer` |
+| E-commerce  | `ecommerce`, `e-commerce`                |
+| Performance | `performance`, `perf`                    |
+| SEO         | `seo`, `performance`, `perf`             |
+
+## Compartilhe
+
+Nós encorajamos que você compartilhe o seu trabalho, assim como amamos ver o que nossos talentosos Astronautas criaram. Venha e compartilhe o que você criou conosco em nosso [Discord](https://astro.build/chat/) ou mencione [@astrodotbuild](https://twitter.com/astrodotbuild) em um Tweet!
```
Comment (victor0x16) [inline src/pages/pt-BR/guides/publish-to-npm.md:282]:
```suggestion
Nós incentivamos que você compartilhe o seu trabalho, assim como amamos ver o que nossos talentosos Astronautas criaram. Venha e compartilhe o que você criou conosco em nosso [Discord](https://astro.build/chat/) ou mencione [@astrodotbuild](https://twitter.com/astrodotbuild) em um Tweet!
```

**Review by victor0x16 — CHANGES_REQUESTED** [review]

**Comment by Yan-Thomas** [comment]
Thanks @victor0x16!

**Review by victor0x16 — APPROVED** [review]
LGTM

**Comment by sarah11918** [comment]
@Yan-Thomas can you take a look at the conflicts here? with nav.ts?

**Comment by Yan-Thomas** [comment]
Solved! @sarah11918

**Pull request merged**